Makefile

```diff
@@ -155,6 +156,12 @@ oc-commands: commands ## Generate oc plugins and add them to build folder
 install-commands: commands ## Generate plugins and add them to /usr/bin/
 	sudo cp -a ./build/. /usr/bin/
 
+.PHONY: tar-commands
+tar-commands: clean ## Generate tar.gz containing krew plugin
+	$(MAKE) KREW_PLUGIN=true kubectl-commands
+	tar -czf netobserv-cli.tar.gz LICENSE ./build/netobserv
+	sha256sum netobserv-cli.tar.gz
```

Comment:
not clear to me how the above tar ball will include krew plugin ?

Reply:
added a script to generate index according to version https://github.com/netobserv/network-observability-cli/pull/27/commits/ad143ad2ad10d6efca69ea037282091936f102fb